source/_components/water_heater.econet.markdown

```diff
@@ -8,7 +8,7 @@ comments: false
 sharing: true
 footer: true
 logo: econet.png
-ha_category: Climate
+ha_category: Water heater
 ha_release: 0.61.0
 ha_iot_class: "Cloud Polling"
 ---
```

Comment:
this needs a redirect header. see docs: [Renaming Pages](https://developers.home-assistant.io/docs/en/documentation_standards.html#renaming-pages)